### PR TITLE
[16.0][MIG] product_total_weight_from_packaging: Migrate to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

**Review by santostelmo — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]

**Comment by rafaelbn** [comment]
@moduon testit @fcvalgar OCA contribution

**Comment by rafaelbn** [comment]
@grindtildeath do you want to review this module as contributor? 💚

File: product_total_weight_from_packaging/models/product.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        self, packaging_tuple, qty_per_pkg
+    ):
+        res = {
+            "qty": qty_per_pkg,
+        }
+        if packaging_tuple.is_unit:
+            res["weight"] = self.weight
+        else:
+            packaging = self.env["product.packaging"].browse(packaging_tuple.id)
+            res["weight"] = packaging.max_weight
```
Comment (jbaudoux) [inline product_total_weight_from_packaging/models/product.py]: The right field is weight in v16. Relates to https://github.com/OCA/product-attribute/pull/1386

```suggestion
            res["weight"] = packaging.weight
```


Also you need to convert the weight in the uom of the product as you can now define a different weight uom on the packaging. So if the product weight uom is `g` as defined on the product and the product packaging weight is `kg` as defined on the product packaging, you need to convert the weight in `kg` to a weight in `g`.

Please fix those 2 things in 2 separate commits

**Review by jbaudoux — CHANGES_REQUESTED** [review]

File: product_total_weight_from_packaging/models/product.py (hunk 1 above)
Comment (QuocDuong1306) [reply product_total_weight_from_packaging/models/product.py]: thank @jbaudoux , I updated

**Review by jbaudoux — APPROVED** [review]
Thanks for the fixes. LGTM

**Pull request merged**